int64 idRobot
int64 idAnimation
string dateStart
string dateEnd